packages/palette/src/elements/Image/LazyImage.tsx

```diff
@@ -9,9 +9,13 @@ import {
 } from "styled-system"
 import { color } from "../../helpers/color"
 import { Box, BoxProps } from "../Box"
-import { Tag } from "../Tag"
+import { omitProps, Tag } from "../Tag"
 import { BaseImage as Image, ImageProps } from "./Image.shared"
 
+const imagePropsToOmit = omitProps.filter(
+  prop => prop !== "width" && prop !== "height"
```

Comment:
Why is `width` and `height` blacklisted?

Reply:
Basically it takes all styled-systems props and removes them. So if width and height were specified to be added as a styled-systems thing you probably down want them to flow down as top level props to the child component. In this case I did.